nengo/processes.py

```diff
@@ -98,14 +98,18 @@ def __repr__(self):
         return "%s(synapse=%r, dist=%r, scale=%r)" % (
             type(self).__name__, self.synapse, self.dist, self.scale)
 
-    def make_step(self, shape_in, shape_out, dt, rng):
+    def allocate(self, shape_in, shape_out, dt, dtype=None):
+        return self.synapse.allocate(shape_out, shape_out, dt, dtype=dtype)
+
+    def make_step(self, shape_in, shape_out, dt, rng, state=None):
         assert shape_in == (0,)
         assert len(shape_out) == 1
 
         dist = self.dist
         scale = self.scale
         alpha = 1. / np.sqrt(dt)
-        filter_step = self.synapse.make_step(shape_out, shape_out, dt, None)
+        filter_step = self.synapse.make_step(
+            shape_out, shape_out, dt, rng=None, state=state)
```

Comment:
Yeah, we had `None` before, too, basically because we assume synapses should not need it. But someone could implement some sort of probabilistic synapse that uses it. I'd make the change, but I'm not sure we need to test it (we have other processes that do use `rng`).